lib/Agrammon/UI/Web.pm6

```diff
@@ -36,52 +35,41 @@ class Agrammon::UI::Web {
         my (@reports, @graphs);
 
         for $!model.evaluation-order -> $module {
-            my @module-inputs;
             for $module.results -> $result {
+                my %result-hash = $result.as-hash;
+
                 my $type = $result.type;
                 my %report = %(
```

Comment:
Missed this before, but if you are assigning pairs into a hash, the `%( ... )` around them is redundant.

Reply:
Thanks, fixed.